dropwizard-app/src/main/java/org/coner/boundary/RegistrationBoundary.java

```diff
@@ -17,6 +17,10 @@ public static RegistrationBoundary getInstance() {
         return instance;
     }
 
+    public static void setInstance(RegistrationBoundary rb) {
```

Comment:
No reason other than I typed it manually instead of generating the setter :).  I will fix.